test/stub-test.js

```diff
@@ -2989,6 +2989,21 @@ describe("stub", function() {
             assert.equals(3, stub.method());
         });
 
+        it("allows providing null as a return value", function() {
+            var Class = function() {
```

Comment:
I am surprised that you're able to use `Class` as a variable name!
It doesn't look right to me ... would you mind renaming it to something that doesn't use a reserved word? ... like `Klass`?

Reply:
Do you mind if I rename it to `TestClass` for all cases in this file?